tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -218,6 +218,17 @@ static PyObject * THPVariable_numel(PyObject* self, PyObject* args)
    END_HANDLE_TH_ERRORS
 }
 
+static PyObject * THPVariable_is_conj(PyObject* self, PyObject* args)
```

Comment:
this and some other changes can probably be skipped now by setting `manual_cpp_binding=True` in `native_functions.yaml`